flutter/lib/common/widgets/toolbar.dart

```diff
@@ -399,6 +400,61 @@ Future<List<TToggleMenu>> toolbarDisplayToggle(
               }
             : null));
   }
+  // follow remote cursor
+  if (pi.platform != kPeerPlatformAndroid &&
```

Comment:
1. lower version check
2. in view mode, these two options are sent, but they are unchecked.